Packs/ThreatGrid/Integrations/ThreatGrid/1_8490e145-c880-4802-a995-9ec32e4f82e2

```diff
@@ -0,0 +1 @@
+{"api_version":2,"id":1234,"request_id":"req-7","data":{"items":[{"path":"w.dll","sample_id":"123","severity":100,"aid":1}],"items_per_page":1000}}
```

Comment:
What is this file?

Reply:
It's from the unit test, I deleted this.